src/components/TheDrawer.vue

```diff
@@ -1,7 +1,12 @@
 <script setup lang="ts">
 import { Drawer, DrawerContent } from "@progress/kendo-vue-layout";
 import { useLocalStorage } from "@vueuse/core";
-import { computed } from "vue";
+import { useRouter } from "vue-router";
+
+import { computed, ref } from "vue";
+
+// const router = useRouter();
```

Comment:
router is commented but used in onSelect function.

Reply:
Was commented in streams code, uncommented